src/dragonball/src/vm/aarch64.rs

```diff
@@ -98,7 +98,8 @@ impl Vm {
         );
         self.vcpu_manager()
             .map_err(StartMicroVmError::Vcpu)?
-            .set_reset_event_fd(reset_eventfd);
+            .set_reset_event_fd(reset_eventfd)
+            .map_err(StartMicroVmError::Vcpu)?;
```

Comment:
It's weird to have two same map_err here. How about using Into/From?

Reply:
Hi, @Tim-Zhang !
Thanks for your recommendation. I have raised an issue for this problem, which explains why we use two map_err here. And in the future, we will work on it. 

issue: #4679